### PR TITLE
Fix doc builder

### PR DESCRIPTION
**Pull request by LysandreJik**
Hotfix to fix the doc-builder

### CODE REVIEW

File: .github/workflows/build_documentation.yml [hunk 1]
```diff
@@ -26,7 +27,7 @@ jobs:
         run: |
           sudo apt-get -y update && sudo apt-get install -y libsndfile1-dev
 
-          pip install -e .
+          pip install git+https://${{ secrets.HUGGINGFACE_PUSH }}@github.com/huggingface/doc-builder
```
Comment (sgugger) [inline .github/workflows/build_documentation.yml:30]: Why do you need the secret? The repo is public.

**Review by sgugger — APPROVED** [review]
Thanks for fixing!

File: .github/workflows/build_documentation.yml (hunk 1 above)
Comment (LysandreJik) [reply .github/workflows/build_documentation.yml:30]: Oh right :)

Comment (LysandreJik) [reply .github/workflows/build_documentation.yml:30]: Will update once I validate that it's passing here

**Comment by LysandreJik** [comment]
Passing! Updated and merging

**Pull request merged**